### PR TITLE
Create support for arm64 architecture for docker images.

### PR DESCRIPTION
**Pull request by kfirtoledo**
Create support for arm64 architecture for docker images:
1) Update the GitHub action.
2) Update the makefile to compile images according to the architecture
3) Update the docker file to use the correct binary file.
4) Update the push image command to use the --platform flag for multi-architecture Docker image compilation.
5) Update the iperf3 demo to support arm64 architecture.

### CODE REVIEW

File: demos/iperf3/testdata/manifests/iperf3-client/iperf3-client.yaml
Comment (elevran) [inline demos/iperf3/testdata/manifests/iperf3-client/iperf3-client.yaml:27]: lighter iperf image, I suppose? Or does it support multi-arch which `mlabbe` does not?

File: demos/utils/cloud.py
Comment (elevran) [inline demos/utils/cloud.py:76]: I understand it is needed but I don't understand why GCP creates nodes with `NoSchedule` in the first place. Seems odd...

**Review by elevran — APPROVED** [review]
lgtm, assuming the `fetch-tags: true` was dropped intentionally. If not - please fix before merging.

File: Makefile
Comment (kfirtoledo) [reply Makefile:6]: NO, GOOS/GOARCH  it is for the go compilation and PLATFORMS is for docker build

File: demos/utils/cloud.py
Comment (kfirtoledo) [reply demos/utils/cloud.py:76]: This problem, I saw happen to other customers.

File: demos/iperf3/testdata/manifests/iperf3-client/iperf3-client.yaml
Comment (kfirtoledo) [reply demos/iperf3/testdata/manifests/iperf3-client/iperf3-client.yaml:27]: Yes, It also compiles to arm64

File: .github/workflows/release.yml
Comment (kfirtoledo) [reply .github/workflows/release.yml:17]: It caused me an error when I used it (the release wasn't tested- because this merged after the last release)

Comment (elevran) [reply .github/workflows/release.yml:17]: what was the error? 
I think we need the tags for the LD_FLAGS, otherwise it reports an error for `git describe`.
Per https://github.com/actions/checkout/issues/701#issuecomment-1139627817, an alternative would be to add another step right below the checkout that does `git fetch --tags origin`

**Review by elevran — APPROVED** [review]

**Pull request merged**
